dev/tests/integration/testsuite/Magento/Bundle/_files/fixed_bundle_product_zero_price_option_selection_rollback.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Registry;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\TestFramework\Workaround\Override\Fixture\Resolver;
+
+Resolver::getInstance()->requireDataFixture('Magento/Bundle/_files/multiple_products_rollback.php');
+$objectManager = Bootstrap::getObjectManager();
+$registry = $objectManager->get(Registry::class);
+$productRepository = $objectManager->create(ProductRepositoryInterface::class);
```

Comment:
please use ->get() for ProductRepositoryInterface

Reply:
Done